test/e2e_node/pod_admission_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2enode
+
+import (
+	"context"
+	"github.com/onsi/ginkgo"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+)
+
+var _ = SIGDescribe("PodOSRejection [NodeConformance]", func() {
```

Comment:
So this is tagged as a `[NodeConformance]` test, running as a node_e2e as opposed to a full cluster e2e.

Node_e2e tests do not use a full cluster and always run on a single node. Hence, `findLinuxNode` is redundant, as this will always run on a single node; either it's Linux, or the test will be skipped. See https://github.com/kubernetes/kubernetes/blob/cd6af9e64b65cb198d633f7c161942b5b9fa155b/test/e2e_node/util.go#L247-L252

I _think_ what you actually want is to run this as a full-cluster e2e, i.e. add the test under `test/e2e/node`.

Reply:
I don't see this getting triggered for the `pull-kubernetes-e2e-aks-engine-windows-containerd` test pass even though `NodeConformance` is in the ginkgo.focus for this test.

I'm not familiar with node_e2e test cases. Are they included in `e2e.test`?

Reply:
@ravisantoshgudimetla can you move the test under `test/e2e/node`?